### ISSUE TITLE
Default file manager options not in settings

### ISSUE THREAD

**Issue by camiloij** [files-community/Website#164]
Hi Files team,

I installed Files app on a Dell XPS 15 and going into the experimental settings the "Default file manager" options is not in the menu.

I have unninstalled and reinstall the app making sure I have the latest update but still the no change.

![No Setting](https://user-images.githubusercontent.com/94451260/155837557-2760eb0f-aefd-4f50-bfdb-9c4c036e95f2.png)
![Version](https://user-images.githubusercontent.com/94451260/155837559-33afe2c8-524e-4a6d-a4ba-45d16ce9c7eb.png)

**Comment by TheOnlyTails** [comment]
This is an issue for the app repo, not the website repo, and yet - the option was removed because there was no way to revert it when uninstalling the app.

**Comment by camiloij** [comment]
The option still appears in my desktop and secondary laptop

**Comment by camiloij** [comment]
All working now. Thank you for your quick answer 👍

**Comment by camiloij** [comment]
Yes, that's what I did after you mentioned the website repo. It work like a charm :) Thanks
